### DISCUSSION TITLE
How to speed up audio (rubberband) without changing the playback speed of the video?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14623] [Q&A]
Hi, I want to change the speed of the audio without affecting the video playback speed. Is this possible?

Thank you :)

**Comment by Dudemanguy** [comment]
Do you mean that you want to change the audio's pitch? That is not implemented but https://github.com/mpv-player/mpv/pull/11214 adds the feature. Or did you want audio to actually to speed up and play ahead of the video?

**Reply by geextahslex** [reply]
The latter. So kinda desync video and audio, so audio can do it's own thing without changing the video playback speed.
I have an external audio track that is too slow, so I need to speed it up to match the video.

**Reply by llyyr** [reply]
https://ffmpeg.org/ffmpeg-filters.html#toc-atempo
